newlib/build-riscv.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+NEWLIB_SRC_DIR=$1
+
+$NEWLIB_SRC_DIR/configure --target=riscv64-unknown-elf \
```

Comment:
why is this riscv64?

Reply:
Because gcc is not user friendly? I have no idea. These are the library versions that are generated:

```
drwxr-xr-x   3 bradjc  staff    96B Aug 11 10:34 rv32i
drwxr-xr-x   3 bradjc  staff    96B Aug 11 10:35 rv32iac
drwxr-xr-x   3 bradjc  staff    96B Aug 11 10:34 rv32im
drwxr-xr-x   3 bradjc  staff    96B Aug 11 10:35 rv32imac
drwxr-xr-x   3 bradjc  staff    96B Aug 11 10:35 rv32imafc
drwxr-xr-x   3 bradjc  staff    96B Aug 11 10:36 rv64imac
```

Reply:
I have an explanation in #231. Basically the tuple indicates only the default compiler configuration.